presto-native-execution/presto_cpp/presto_protocol/presto_protocol.yml

```diff
@@ -82,13 +84,15 @@ AbstractClasses:
       subclasses:
         - { name: HiveTransactionHandle,    key: hive }
         - { name: RemoteTransactionHandle,  key: $remote }
+        - { name: SystemTransactionHandle,  key: $system }
 
     ConnectorTableLayoutHandle:
       super: JsonEncodedSubclass
       subclasses:
         - { name: HiveTableLayoutHandle,    key: hive }
         - { name: IcebergTableLayoutHandle, key: hive-iceberg }
         - { name: TpchTableLayoutHandle,    key: tpch }
+        - { name: SystemTableLayoutHandle,  key: $system }
```

Comment:
Why don't we need `system` and `$system@system` here?

Reply:
So we need the protocol json classes to be generated only once in this script. There isn't a need for all 3 catalog name mappings here. The mapping of the protocol to the key/catalog name happens in the PrestoToVeloxConnector code now. So that's where we have the 3 catalog name mappings.